lib/rules/check-require.js

```diff
@@ -3,10 +3,27 @@
 const path = require('path')
 const Module = require('module')
 
+let modules = Object.keys(process.binding('natives'))
+const internalModules = new Set()
+const UNSAFE_MODULE = /^internal/
+
+for (const name of modules) {
+  if (UNSAFE_MODULE.test(name)) continue
+  internalModules.add(name)
+  internalModules.add(`node:${name}`)
```

Comment:
I wonder if instead of doubling this list, the `check` below should be splitting the name on `:` and treating the existence of a `node` namespace as a signal to _only_ consider builtins.

`Module.isBuiltin` (when available) will work with with/without the namespace